### PR TITLE
[conda] fixed service component bug

### PR DESCRIPTION
**Pull request by lamarrr**

### CODE REVIEW

**Comment by lamarrr** [comment]
> Nice! Is there any way you could easily add a test that would have failed here and caught this? 🙏🏼
I thought so too.
I couldn't find any test suites for the origin components doing that

**Comment by lumaxis** [comment]
@lamarrr Added a quick file to do that: https://github.com/lamarrr/clearlydefined-service/pull/1 Let me know what you think!

**Comment by lamarrr** [comment]
LGTM, thanks for the help!

**Review by lumaxis — APPROVED** [review]

File: routes/originConda.js [hunk 1]
```diff
@@ -35,66 +34,75 @@ async function fetchCondaRepoData(channel, subdir) {
...
-          if (packageData.name === name) {
-            revisions.push(`${subdir}:${packageData.version}-${packageData.build}`)
-          }
-        })
-      }
-    }
-    return response.status(200).send(uniq(revisions))
-  })
-)
+router.get('/:channel/:subdir/:name/revisions', asyncMiddleware(getOriginCondaRevisions()))
```
Comment (qtomlinson) [inline routes/originConda.js:37]: take out the (), function not function invocation: `asyncMiddleware(getOriginCondaRevisions)`

**Review by qtomlinson — COMMENTED** [review]
Great to see the collaborative effort!

File: routes/originConda.js (hunk 1 above)
Comment (qtomlinson) [reply routes/originConda.js:37]: Fixed by https://github.com/clearlydefined/service/pull/1113

**Pull request merged**